start

```diff
@@ -181,7 +245,47 @@ load(
     "rules_haskell_toolchains",
 )
 
+# Download alex and make it accessible as @alex
+http_archive(
+    name = "alex",
+    # Specify to build alex using cabal:
+    build_file_content = """
+load("@rules_haskell//haskell:cabal.bzl", "haskell_cabal_binary")
+haskell_cabal_binary(
+    name = "alex",
+    srcs = glob(["**"]),
+    visibility = ["//visibility:public"],
+)
+    """,
+    sha256 = "d58e4d708b14ff332a8a8edad4fa8989cb6a9f518a7c6834e96281ac5f8ff232",
+    strip_prefix = "alex-3.2.4",
+    urls = ["http://hackage.haskell.org/package/alex-3.2.4/alex-3.2.4.tar.gz"],
+)
+
+# Download alex input file and make it accessible as @alex_example//file
+http_file(
+    name = "alex_example",
+    downloaded_file_path = "words.x",
+    sha256 = "2fb093eb46385e61abbfffa9ccc675b4cda3775d205f3da1917d33ab4e21ed0c",
+    urls = ["https://raw.githubusercontent.com/simonmar/alex/60d59327fdebc98ea1d8738ce07c174ba7e76f23/examples/words.x"],
+)
+
+load(
+    "@rules_haskell//haskell:cabal.bzl",
+    "stack_snapshot"
+)
+
+stack_snapshot(
+    name = "stackage",
+    extra_deps = {"zlib": ["@zlib.dev//:zlib"]},
+    packages = ["zlib"],
+    snapshot = "lts-14.27" # Last snapshot published for ghc-8.6.5
+    # the default version picked up by rules_haskell
+)
```

Comment:
Sorry, I should have been more clear what I meant with the Alex example. What I had in mind was to use it as a `tool` in `stack_snapshot`, like we do in `rules_haskell`: https://github.com/tweag/rules_haskell/blob/827d654235189242b0a9d45df8bc8754469b4579/WORKSPACE#L105-L108

Such tool dependencies are relatively common as a project's dependency graph grows. So, it seems reasonable to include an example in the `start` script to avoid people getting stuck when they reach that point.

Reply:
Issue addressed in 8f723ad7bd326ddadea066e252385c1dafa527a8